backend/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointEventTypeRepository.java

```diff
@@ -107,6 +107,54 @@ public void updateEventTypesLinkedToEndpoint(UUID endpointId, Set<UUID> eventTyp
         entityManager.merge(endpoint);
     }
 
+    @Transactional
+    public void refreshEndpointLinksToEventType(String orgId, List<UUID> endpointsList) {
+        if (endpointsList == null || endpointsList.isEmpty()) {
+            return;
+        }
+
+        String deleteQueryStr = "DELETE FROM EndpointEventType eet WHERE " +
+            "endpoint.orgId " + (orgId == null ? "is null " : "=: orgId ") +
```

Comment:
Does Postgres understand `=: orgId` as `= :orgId`?

Reply:
I tested it properly and it works, but it wasn't on purpose.
I added a new commit to use our usual syntax.